### PR TITLE
[13.0][MIG] account_invoice_search_by_reference

### PR DESCRIPTION
**Pull request by newtratip**
This module is search by name or reference in the invoice.
Peek 2020-03-12 15-26

### CODE REVIEW

File: account_invoice_search_by_reference/__manifest__.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Account invoice search by reference",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "author": "Eficent, Odoo Community Association (OCA)",
     "category": "Invoicing",
     "website": "http://www.eficent.com/",
```
Comment (nikul-serpentcs) [inline account_invoice_search_by_reference/__manifest__.py]: @newtratip  URL should be https://github.com/OCA/account-invoicing

File: account_invoice_search_by_reference/model/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
```
Comment (AaronHForgeFlow) [inline account_invoice_search_by_reference/model/account_invoice.py]: Thanks for migrating this PR. Could you change the copyright please? Now it is

`Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)`

File: account_invoice_search_by_reference/__manifest__.py (hunk 1 above)
Comment (newtratip) [reply account_invoice_search_by_reference/__manifest__.py]: @nikul-serpentcs 
Edited, Thank you for your review.

File: account_invoice_search_by_reference/model/account_invoice.py (hunk 2 above)
Comment (newtratip) [reply account_invoice_search_by_reference/model/account_invoice.py]: @AaronHForgeFlow 
Edited, Thank you for your review.

File: account_invoice_search_by_reference/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Account invoice search by reference",
+    "version": "13.0.1.0.0",
+    "author": "Eficent, Odoo Community Association (OCA)",
```
Comment (AaronHForgeFlow) [inline account_invoice_search_by_reference/__manifest__.py]: Sorry, can you change also Eficent by ForgeFlow? Thanks again.

**Review by AaronHForgeFlow — COMMENTED** [review]
CI is red

File: account_invoice_search_by_reference/models/account_invoice.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models
+
+
+class AccountInvoice(models.Model):
```
Comment (kittiu) [inline account_invoice_search_by_reference/models/account_invoice.py]: In v13, there is no longer file named "accont_invoice.py", it should be account_move.py.
Also no `class AccountInvoice`, should be `class AccountMove`

Comment (kittiu) [reply account_invoice_search_by_reference/models/account_invoice.py]: @newtratip please fix it.

Comment (kittiu) [reply account_invoice_search_by_reference/models/account_invoice.py]: @newtratip can you attend this?

File: account_invoice_search_by_reference/__manifest__.py (hunk 3 above)
Comment (AaronHForgeFlow) [reply account_invoice_search_by_reference/__manifest__.py]: Thank you! will do a functional review

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests on runbot. Could you please attend the other reviews?

**Comment by newtratip** [comment]
@AaronHForgeFlow 

Yes, I will attend the other reviews.

Thank for your review :)

File: account_invoice_search_by_reference/models/account_invoice.py (hunk 4 above)
Comment (newtratip) [reply account_invoice_search_by_reference/models/account_invoice.py]: Fixed, please review again.

Thanks.

**Review by ps-tubtim — APPROVED** [review]
LGTM

**Pull request merged**